### PR TITLE
Add check to cli 

### PR DESCRIPTION
**Pull request by grimadas**
Add checks to cli to check if file defined in the flags is present.

### CODE REVIEW

File: iroha-cli/main.cpp [hunk 1]
```diff
@@ -82,13 +82,19 @@ int main(int argc, char* argv[]) {
       logger->info("Send transaction to {}:{} ", FLAGS_address,
                    FLAGS_torii_port);
       std::ifstream file(FLAGS_json_transaction);
+      if (not file.is_open()){
```
Comment (lebdron) [inline iroha-cli/main.cpp]: Execution should probably stop if such error occurs.

**Comment by neewy** [comment]
It was proposed to introduce mechanism of handling critical situations both in irohad, iroha-cli. Consider changing this PR after we will decide on that

File: iroha-cli/main.cpp [hunk 2]
```diff
@@ -91,6 +92,10 @@ int main(int argc, char *argv[]) {
```
Comment (x3medima17) [inline iroha-cli/main.cpp]: curl

Comment (x3medima17) [inline iroha-cli/main.cpp]: test

**Pull request closed without merging**